src/ErrorHandler.php

```diff
@@ -434,16 +429,18 @@ private function handleException(\Throwable $exception): void
         try {
             if (null !== $previousExceptionHandler) {
                 $previousExceptionHandler($exception);
+
+                return;
             }
         } catch (\Throwable $previousExceptionHandlerException) {
-            // Do nothing, we just need to set the $previousExceptionHandlerException
-            // variable to the exception we just catched to compare it later
-            // with the original object instance
+            // This `catch` statement seems to be useless, but in reality its purpose
+            // is to set the $previousExceptionHandlerException variable to the exception
+            // we just catched
         }
 
-        // If the exception object instance is the same as the one catched from
-        // the previous exception handler, if any, give it back to the native
-        // PHP handler to prevent infinite circular loop
+        // If the instance of the exception we're handling is the same as the one
+        // catched from the previous exception handler then we give it back to the
+        // native PHP handler to prevent infinite circular loop
```

Comment:
```suggestion
        // If the instance of the exception we're handling is the same as the one
        // catched from the previous exception handler then we give it back to 
        // the native PHP handler to prevent an infinite loop
```